.github/workflows/build-images-workflow-run.yml

```
@@ -66,7 +66,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           sourceRunId: ${{ github.event.workflow_run.id }}
       - name: "Cancel duplicated 'CI Build' runs"
-        uses: potiuk/cancel-workflow-runs@99869d37d982384d18c79539b67df94f17557cbe  # v4_1
+        uses: potiuk/cancel-workflow-runs@19f8bd20fd32b5b1d7857036b98e62550a442616  # v4_2
```

```suggestion
        uses: potiuk/cancel-workflow-runs@19f8bd20fd32b5b1d7857036b98e62550a442616  # v4_3
```
Should`4_3` instead of `4_2` be here?

---

Yeah. But I just released 4.4 with self-preservation mechanism so it is outdated :)